<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GIScan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="maplayout"/>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="graphlayout"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="5" column="0" alignment="Qt::AlignTop">
         <widget class="QPushButton" name="holdVertical">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>218</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Hold vertical</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignTop">
         <widget class="QPushButton" name="saveVertical">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Save vertical cut as txt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignTop">
         <widget class="QPushButton" name="load_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Load Data</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="holdHorizontal">
          <property name="text">
           <string>Hold Horizontal</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="saveHorizontal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Save horizontal cut as txt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionOpen_SampleDB">
   <property name="text">
    <string>Open SampleDB</string>
   </property>
   <property name="shortcut">
    <string comment="test" extracomment="test"/>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOff_specular">
   <property name="text">
    <string>Off-specular</string>
   </property>
  </action>
  <action name="actionNeutrons">
   <property name="text">
    <string>Neutrons</string>
   </property>
  </action>
  <action name="actionX_Rays">
   <property name="text">
    <string>X-Rays</string>
   </property>
  </action>
  <action name="actionOpen_single_specular_file">
   <property name="text">
    <string>Open single specular file</string>
   </property>
  </action>
  <action name="actionSpecular_X_ray">
   <property name="text">
    <string>Specular X-ray</string>
   </property>
  </action>
  <action name="menuXraySource">
   <property name="text">
    <string>X-rays</string>
   </property>
  </action>
  <action name="menuNeutronSource">
   <property name="text">
    <string>Neutrons</string>
   </property>
  </action>
  <action name="actionSpecular_neutron">
   <property name="text">
    <string>Specular neutron</string>
   </property>
  </action>
  <action name="actionOff_specular_neutron">
   <property name="text">
    <string>Off-specular neutron</string>
   </property>
  </action>
  <action name="actionData_tools">
   <property name="text">
    <string>General analysis</string>
   </property>
  </action>
  <action name="actionDetect_peaks">
   <property name="text">
    <string>Detect peaks</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
